frappe/database/postgres/database.py

```diff
@@ -169,6 +170,15 @@ def setup_type_map(self):
 	def last_query(self):
 		return LazyDecode(self._cursor.query)
 
+	@property
+	def db_schema(self):
+		return re.sub(r'["\']', "", frappe.conf.get("db_schema", "public"))
```

Comment:
Any specific region to use regex here? Should be fine with just `.replace()` right? (mainly: more readable)

Reply:
Sure. Its just 2 replaces chained then.